src/main/java/com/sk89q/jinglenote/bukkit/BukkitJingleNotePlayer.java

```diff
@@ -31,22 +31,20 @@ public void play (Note note)  {
 
     public Sound toSound(Instrument instrument) {

```

Comment:
You can entirely strip out JingleNote. It's an external library now (and sounds much better)